recipe_scrapers/aldinord.py

```diff
@@ -0,0 +1,20 @@
+from ._abstract import AbstractScraper
+
+
+class AldiNord(AbstractScraper):
+    @classmethod
+    def host(cls, domain: str = "aldi-nord.de"):
+        return domain
+
+    def author(self):
+        return "ALDI"
```

Comment:
Suggestion 2 of 2: these recipe pages seem to have `schema.org` `Recipe` info on the page -- but the `author` field is the empty string.

I think `ALDI` is a good fallback value here, but if the website does begin populating the recipe author field, I think that'd be preferable.  So I'd suggest something like:

```suggestion
    def author(self):
        if author_from_schema := self.schema.author():
            return author_from_schema
        return "ALDI"
```

Reply:
Or just `return self.schema.author() or "ALDI"` or I'm missing something with these := 🤔

Reply:
(NB: when Python 3.11+ becomes our support baseline, I think it may be possible to replace that exception-based interface -- it seems non-Pythonic -- with [`LiteralString` typing](https://docs.python.org/3/library/typing.html#typing.LiteralString) instead.  I'm not 100% sure about that yet, but I'm optimistic)

Reply:
Thanks for the comments. I took both